ext/fg/js/source.js

```diff
@@ -117,17 +118,19 @@ class TextSourceRange {
 
             if (nodeType === TEXT_NODE) {
                 state.node = node;
-                if (resetOffset) {
-                    state.offset = forward ? 0 : node.nodeValue.length;
-                }
-                if (lineBreak) {
-                    if (resetOffset && addLineBreak(state, forward)) {
+                if (!resetOffset || shouldSeekTextNode(node)) {
```

Comment:
For some reason, `shouldSeekTextNode` gets an off-screen ARIA element on Google Docs, returns `false` and after that `Range.setEnd` errors like this:

```core.js:81 Yomichan v1.9.5 has encountered an error.
Originating URL: https://docs.google.com/document/d/1GvxFCXYpAFte9_qEwg2fCfFP0HvX68Dee3RmHAR2MAU/edit
IndexSizeError: Failed to execute 'setEnd' on 'Range': The offset 50 is larger than the node's length (28).
Error: Failed to execute 'setEnd' on 'Range': The offset 50 is larger than the node's length (28).
    at TextSourceRange.setEndOffset (chrome-extension://jpjldhmgaijbpomlojkhkghbbebbiblb/fg/js/source.js:52:20)
    at docSentenceExtract (chrome-extension://jpjldhmgaijbpomlojkhkghbbebbiblb/fg/js/document.js:148:17)
    at Frontend.showContent (chrome-extension://jpjldhmgaijbpomlojkhkghbbebbiblb/fg/js/frontend.js:395:26)
    at Frontend.searchSource (chrome-extension://jpjldhmgaijbpomlojkhkghbbebbiblb/fg/js/frontend.js:368:26)
    at async Frontend.searchAt (chrome-extension://jpjldhmgaijbpomlojkhkghbbebbiblb/fg/js/frontend.js:345:17)
    at async search (chrome-extension://jpjldhmgaijbpomlojkhkghbbebbiblb/fg/js/frontend.js:104:13)

Issues can be reported at https://github.com/FooSoft/yomichan/issues```

Reply:
This issue has been fixed in 047cfba227f7f01a34763ffe6867842662cc02b5, but it has brought another issue to my attention. Will document it in a separate comment.